<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationTool</class>
 <widget class="QMainWindow" name="ClassificationTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>698</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="DrawWidget" name="Section" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1200</width>
            <height>600</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>600</width>
            <height>300</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QPushButton" name="Save">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lenght_section">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>20</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QToolButton" name="RectangleSelection">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Grafiken/Rechteck.jpg</normaloff>Grafiken/Rechteck.jpg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="Previous">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::LeftArrow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="Next">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::RightArrow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="ClassList"/>
          </item>
          <item row="0" column="5">
           <widget class="QToolButton" name="PointSelection">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Grafiken/Zeiger.png</normaloff>Grafiken/Zeiger.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="width_section">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="length">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>Lenght</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="width">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="Elevator">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="QLabel" name="loadFile">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Pointcloud:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout"/>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="LoadButton">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Pointcloud</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="LoadAxis">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>56</width>
                <height>19</height>
               </size>
              </property>
              <property name="text">
               <string>Axis</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>19</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>50</width>
                <height>19</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>50</width>
                <height>19</height>
               </size>
              </property>
              <property name="text">
               <string>Axis:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="PathToAxisShp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>291</width>
                <height>19</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="PathToFile">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="ClassifiedPoints">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>DrawWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
